pyproject.toml

```diff
@@ -53,44 +53,44 @@ sphinx-rtd-theme = "^1.0.0"
 
 [tool.ruff]
 fix = true
+unfixable = [
+    "ERA", # do not autoremove commented out code
+]
 target-version = "py37"
 line-length = 88
 extend-select = [
-    "B",  # flake8-bugbear
+    "B",   # flake8-bugbear
     "C4",  # flake8-comprehensions
-    "ERA",  # flake8-eradicate/eradicate
-    "PIE",  # flake8-pie
-    "SIM",  # flake8-simplify
-    "TID",  # flake8-tidy-imports
-    "TCH",  # flake8-type-checking
-    "N",  # pep8-naming
-    "RUF",  # ruff checks
+    "ERA", # flake8-eradicate/eradicate
+    "I",   # isort
+    "N",   # pep8-naming
+    "PIE", # flake8-pie
+    "PGH", # pygrep
+    "RUF", # ruff checks
+    "SIM", # flake8-simplify
+    "TCH", # flake8-type-checking
+    "TID", # flake8-tidy-imports
+    "UP",  # pyupgrade
+]
+ignore = [
+    "B904", # use 'raise ... from err'
+    "B905", # use explicit 'strict=' parameter with 'zip()'
+    "N818", #  Exception name should be named with an Error suffix
```

Comment:
This shouldn't be needed, since cleo has `Error` suffixes. Please check other ignored rules if they are actually needed.

Reply:
Good call, none were needed